lms/djangoapps/courseware/tests/test_utils.py

```diff
@@ -127,21 +127,7 @@ def test_get_financial_assistance_application_status_unsuccessful(self, response
             assert has_application is False
             assert reason == response_data.get('content').get('message')
 
-    @ddt.data(
-        {
-            'status': status.HTTP_400_BAD_REQUEST,
-            'content': {'message': 'Invalid course id provided'},
-            'message': 'Invalid course id provided',
-            'created': False
-        },
-        {
-            'status': status.HTTP_200_OK,
-            'content': {'success': True},
-            'message': None,
-            'created': True
-        }
-    )
```

Comment:
why is this removed?

Reply:
The usecase for `is_eligible` was altered hence I changed the tests too.